x/ibc/04-channel/client/cli/query.go

```diff
@@ -203,27 +204,28 @@ func GetCmdQueryPacketCommitment(clientCtx client.Context) *cobra.Command {
 
 // GetCmdQueryUnrelayedPackets defines the command to query all the unrelayed packets.
 func GetCmdQueryUnrelayedPackets(clientCtx client.Context) *cobra.Command {
-	return &cobra.Command{
-		Use:   "unrelayed-packets [port-id] [channel-id] [sequences]",
+	cmd := &cobra.Command{
+		Use:   "unrelayed-packets [port-id] [channel-id]",
 		Short: "Query all the unrelayed packets associated with a channel",
-		Long: `Query all the unrelayed packets associated with a channel.
-The sequences argument is a comma separated list of numbers.`,
-		Example: fmt.Sprintf("%s query %s %s unrelayed-packets [port-id] [channel-id]", version.ClientName, host.ModuleName, types.SubModuleName),
-		Args:    cobra.ExactArgs(3),
-		RunE: func(_ *cobra.Command, args []string) error {
+		Long: `It indicates if a packet, given a list of packet commitment sequences, is unrelayed.
+An unrelayed packet corresponds to:
+
+- Unrelayed packet commitments: when no acknowledgement exists for the given sequence.
+- Unrelayed packet acknowledgements: when an acknowledgement exists and a packet commitment also exists.`,
+		Example: fmt.Sprintf("%s query %s %s unrelayed-packets [port-id] [channel-id] --sequences=1,2,3", version.ClientName, host.ModuleName, types.SubModuleName),
+		Args:    cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx = clientCtx.Init()
 			queryClient := types.NewQueryClient(clientCtx)
 
-			seqStrs := strings.Split(args[2], ",")
-
-			seqs := make([]uint64, len(seqStrs))
+			seqSlice, err := cmd.Flags().GetInt64Slice(flagSequences)
```

Comment:
ah I meant if the sequence flag is unused that the default behaviour is to query for the packet commitments and supply them as the sequence arg. This can be done in a followup pr though